src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -484,6 +502,56 @@ async def _genomic_to_transcript_exon_coordinate(
 
         params = {key: None for key in TranscriptExonData.model_fields}
 
+        if is_fusion_transcript_segment:
+            # Check if is_fusion_transcript_segment is given
+            alt_ac = self.sr.chromosome_to_acs(chromosome)[0][0]
+            if alt_ac is None:
+                return self._return_warnings(
+                    resp, "Chromosome could not be converted to an accession"
+                )
+            mane_transcripts = self.mane_transcript_mappings.get_gene_mane_data(gene)
+            if mane_transcripts:
+                transcript = mane_transcripts[0]["RefSeq_nuc"]
+
+            else:
+                result = await self.uta_db.get_transcripts(gene=gene, alt_ac=alt_ac)
+                if len(result) > 0:
+                    transcript = result[0]["tx_ac"][0]
```

Comment:
```suggestion
                results = await self.uta_db.get_transcripts(gene=gene, alt_ac=alt_ac)
                if results:
                    transcript = results[0]["tx_ac"][0]
```

Reply:
@korikuzma Result can return a dataframe with 0 rows, and 4 columns, so would I have to use `len` since it would return a non-empty dataframe?

Reply:
`df.is_empty()` is what you can check for empty df